### DISCUSSION TITLE
How to disable going in Visual Mode when selecting keyword using multi-cursor (cmd-d)?

### DISCUSSION THREAD

**Discussion by saneef** [NeoVintageous/NeoVintageous#986] [Q&A] [answered]
Without NeoVintageous (and I believe in earlier versions of NV): https://d.pr/v/1odSmP
In NeoVintageous v1.33 it goes into : https://d.pr/v/Lz171k

Is there a way to disable this behaviour?

**Comment by gerardroche** [comment, marked as answer]
Fixed in 1.33.1. Re https://github.com/NeoVintageous/NeoVintageous/issues/985
